node/events-to-ga/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "events-to-ga",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/main.js",
+  "type": "module",
+  "scripts": {
+    "format": "prettier --write ."
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "node-appwrite": "^11.0.0",
+    "node-fetch": "^3.3.2",
```

Comment:
For consistency, please use undici like the other templates:

https://github.com/appwrite/templates/blob/df0b4ef612f54bbe8b3b5cf3f099137d37952ee8/node/analyze-with-perspectiveapi/package.json#L12